patient-appointment-management/assets/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Patient Appointment Management Application</title>
+  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+  <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+  <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+  <link rel="stylesheet" href="/styles.css">
+</head>
+<body>
+  <header>
+    <h1>Patient Appointment Management Application</h1>
+    <div id="hipaa-disclaimer" style="margin-bottom: 20px;">
+        Note that this application is designed for prototyping purposes only and you should<br/>
```

Comment:
Possibly a good paragraph to include in `README.md` as well.

Reply:
Yes and in the customer IG.